Dispatcher/Supporting Files/Constants.swift

```diff
@@ -37,4 +37,16 @@ struct Constants {
         static let appSettingSection = "SettingSectionCell"
         static let recentSearch = "RecentSearchCell"
     }
+    
+    struct UserDefaults {
+        static let recentSearches = "recentSearches"
+    }
+    
+    struct apiCalls {
+        static let newsUrl = "https://api.newscatcherapi.com/v2/search"
+    }
+    
+    struct Keys {
+        static let newApi = "_E1saiwKuTTM_aofezdXNrGCZsgavQ89t-bJ8Z6WlXA"
```

Comment:
Better to call it "newsApiKey" since it is a key

Reply:
Fixed
@abenartz

Reply:
@noamkuMoDev 
Please comment as fixed after you pushed the changes in order to me to resolve the comment

Reply:
Fixed & Pushed 👍🏻
@abenartz